src/helpers/tokenSectionTypes.ts

```diff
@@ -1,9 +1,9 @@
 export default {
-  bridgeTokenSection: '􀊝 Bridge',
-  crosschainMatchSection: '􀤆 On other networks',
-  favoriteTokenSection: '􀋃 Favorites',
-  lowLiquidityTokenSection: '􀇿 Low Liquidity',
-  unverifiedTokenSection: '􀇿 Unverified',
-  verifiedTokenSection: '􀇻 Verified',
-  unswappableTokenSection: '􀘰 No trade routes',
+  bridgeTokenSection: 'bridgeTokenSection',
```

Comment:
Idk if this is doing a whole lot for us atm, but all good for now! It's got me thinking about how we could strongly type the `lang` util 🤔